### PR TITLE
Features/fix accept explossive button

### PR DESCRIPTION
**Pull request by Yonathan-Admasu728**
This PR  changes the appearance of the Accept mission button:
In this PR I Pair Programed with Tony Miller.  Tony's Role was as a Navigator and mine was as a Driver
./src/components/pages/GamePlay/GamePlayMain.js
line 14: imported the explosion.png from the assets.
line 108 - 110: gave it a classname, gave it alt attribute and used it as img src ="" 
./src/styles/less/Gamemodestyles.less
line 40 - 54: styled it to match the desired specs. using z-index to pop up the button txt with some shadow effect on the explosive-star like background.

Resolves Trello card: https://trello.com/c/eROpAytG/658-gameplay-add-design-vector-to-accept-the-mission-button
PR video walkthrough:https://loom.com/share/d59d4223b60640d7b4055be8f0906cef

### CODE REVIEW

**Review by rhiannonstanford — APPROVED** [review]
Wonderful work! Nice job changing the look of the page to match the Figma.  Thank you!

**Review by artofmayhem — APPROVED** [review]
Well done Yonathan! Nice and clean. Fun time pair coding this with you!

File: src/styles/less/GamemodeStyles.less [hunk 1]
```diff
@@ -33,14 +33,27 @@
       }
 
       &.mission-btn {
-        background: #c6fd7e
+        background: #C6FD7E
```
Comment (Alisa1989) [inline src/styles/less/GamemodeStyles.less:36]: make sure this color is approved:
https://www.figma.com/file/U9Lo9qjZgl4Vbgb2cCMq2v/Story-Squad---Labs?node-id=522%3A1456

**Review by Alisa1989 — CHANGES_REQUESTED** [review]

File: src/styles/less/GamemodeStyles.less (hunk 1 above)
Comment (ghost) [reply src/styles/less/GamemodeStyles.less:36]: The color didn't actually change. The hex code is just capitalized now, but it is in the figma color palette.

Comment (Yonathan-Admasu728) [reply src/styles/less/GamemodeStyles.less:36]: these colors are not part of the changes in this PR.  I have mentioned all lines in each files which are part of the change in this PR

**Review by Alisa1989 — APPROVED** [review]

**Pull request merged**